packages/react-core/src/components/Page/Page.tsx

```diff
@@ -302,6 +305,7 @@ class Page extends React.Component<PageProps, PageState> {
         className={css(styles.pageMain)}
         tabIndex={mainTabIndex}
         aria-label={mainAriaLabel}
+        style={{ width: `${drawerWidth}` }}
```

Comment:
@mcoker do we want to set the ` --pf-v5-c-drawer__panel--md--FlexBasis` css var here instead?

Reply:
I'm actually not sure what this is doing... it's being added to the page's main element inside of drawer content. Here's a screenshot of `drawerWidth` set to 500px. The drawer panel gets the correct width, but you can see the page main element is also 500px - is that intentional?

<img width="1807" alt="Screenshot 2024-04-17 at 3 23 47 PM" src="https://github.com/patternfly/patternfly-react/assets/35148959/8a1e20a7-b2c5-4c85-93e0-fc43502778e1">